File: adm/count_after_ingest.php
<?php
$dsn = "pgsql:host=127.0.0.1;port=5432;dbname=spiv";
$user='postgres'; $pass='LulaTetra26';
try{
    $pdo=new PDO($dsn,$user,$pass,[PDO::ATTR_ERRMODE=>PDO::ERRMODE_EXCEPTION]);
}catch(Exception $e){ echo "CONNECT ERROR: " . $e->getMessage() . PHP_EOL; exit(1); }
$tables=['empresas','estabelecimentos','socios','simples','cnaes','municipios'];
foreach($tables as $t){
    try{
        $stmt=$pdo->query("SELECT COUNT(*) as cnt FROM receita.\"$t\"");
        $cnt=$stmt->fetch(PDO::FETCH_ASSOC)['cnt'];
        echo "$t: $cnt\n";
    }catch(Exception $e){
        echo "$t: ERROR: " . $e->getMessage() . "\n";
    }
}

File: adm/count_after_ingest.py
import psycopg2
cfg = {"host":"localhost","port":5432,"dbname":"spiv","user":"postgres","password":"LulaTetra26"}
conn = psycopg2.connect(**cfg)
cur = conn.cursor()
for t in ['empresas','estabelecimentos','socios','simples','cnaes','municipios']:
    try:
        cur.execute(f"SELECT COUNT(*) FROM receita.{t}")
        print(f"{t}: {cur.fetchone()[0]}")
    except Exception as e:
        print(f"{t}: ERROR: {e}")
cur.close(); conn.close()
